pkg/controller/application/status.go

```diff
@@ -1,5 +1,17 @@
-// Copyright 2020 The Kubernetes Authors.
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
I'm not sure we can replace this with our license header, this was kept intentionally, because this file is copied from `sig.k8s.io/application`. The rest looks good to me.

Reply:
Thanks for the reminder, already ignored.